src/style.css

```diff
@@ -2,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, Arial, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search--big {
+  height: 70px;
```

Comment:
This is a code quality issue. You've fixed the height of the `.search--big` class. Unless it's a specific requirement, it's generally best not to fix the height of containers. Instead, let the content dictate the size to avoid overflow or accidental scroll bars.